gradle-plugin-testing-junit/build.gradle

```diff
@@ -30,6 +30,11 @@ dependencies {
     testFixturesImplementation 'org.junit.jupiter:junit-jupiter-params'
 }
 
+tasks.named('test', Test) {
+    // by default the tests should run with configuration cache enabled.
+    systemProperty 'com.palantir.gradle.testing.configuration_cache_enabled', 'true'
```

Comment:
Once we publish this version, I'll add the `isConfigurationCacheEnabled = true` in the `gradleTestutils` extension